src/utils/common-expression-helpers.ts

```diff
@@ -260,17 +260,7 @@ export class CommonExpressionHelpers {
   };
 
   calcGravida = (parityTerm, parityAbortion) => {
-    let gravida = 0;
-    if (parityTerm === parseInt(parityTerm)) {
-      gravida = parityTerm + 1;
-    }
-    if (parityAbortion === parseInt(parityAbortion)) {
-      gravida = parityAbortion + 1;
-    }
-    if (parityAbortion === parseInt(parityAbortion) && parityTerm === parseInt(parityTerm)) {
-      parityAbortion + parityTerm + 1;
-    }
-    return gravida;
+    return parseInt(parityTerm) + parseInt(parityAbortion);
```

Comment:
So in other words the Gravida is the summation of the `parityTerm` with `parityAbortion`?

Reply:
Affirmative.

Reply:
Per our conversation from Coffee Break call, you might need to consider the current pregnancy in your calculation @arodidev.

Reply:
@denniskigen I did a bit of consultation, and it seems the convention should be to do a summation of the two.